### PR TITLE
Update icons in components to new icon set

### PR DESCRIPTION
**Pull request by thisisdano**
Makes icons more consistent and up-to-date with our current icon set.

[Preview](https://federalist-3b6ba08e-0df4-44c9-ac73-6fc193b0e19c.app.cloud.gov/preview/uswds/uswds/dw-update-icons/) →

#### Old
Screen Shot 2021-02-11 at 12 06 34 PM

#### New
Screen Shot 2021-02-11 at 12 06 54 PM

#### Old
Screen Shot 2021-02-11 at 3 25 23 PM

#### New
Screen Shot 2021-02-11 at 3 25 06 PM

#### Old
Screen Shot 2021-02-12 at 9 28 15 AM

#### New
Screen Shot 2021-02-12 at 9 28 02 AM

#### Old
Screen Shot 2021-02-12 at 9 29 51 AM

#### New
Screen Shot 2021-02-12 at 9 29 36 AM

#### Old
Screen Shot 2021-02-12 at 9 32 14 AM

#### New
Screen Shot 2021-02-12 at 9 31 53 AM

#### Old
Screen Shot 2021-02-12 at 9 41 15 AM

#### New
Screen Shot 2021-02-12 at 9 42 27 AM

#### Old
Screen Shot 2021-02-12 at 9 44 12 AM

#### New
Screen Shot 2021-02-12 at 9 44 27 AM

#### Old
Screen Shot 2021-02-12 at 9 46 14 AM

#### New
Screen Shot 2021-02-12 at 9 46 28 AM

#### Old
header-nav-button--old

#### New
header-nav-button--new

#### Old
Screen Shot 2021-02-12 at 10 34 50 AM

#### New
Screen Shot 2021-02-12 at 10 34 40 AM

### CODE REVIEW

File: src/stylesheets/elements/form-controls/_combo-box.scss [hunk 1]
```diff
@@ -42,25 +42,26 @@ button.usa-combo-box__clear-input {
 .usa-combo-box__clear-input {
   background-color: transparent;
   background-position: center;
-  background-size: auto units(1.5);
```
Comment (mejiaj) [inline src/stylesheets/elements/form-controls/_combo-box.scss:45]: We might need to tweak the dimensions of the inputs.

File: src/stylesheets/elements/form-controls/_date-picker.scss [hunk 2]
```diff
@@ -45,7 +45,7 @@
 
 .usa-date-picker__button {
   @extend %usa-date-picker__button;
-  @include add-background-svg("calendar-alt-solid");
+  @include add-background-svg("usa-icons/calendar_today");
```
Comment (mejiaj) [inline src/stylesheets/elements/form-controls/_date-picker.scss:48]: Height seems to be smaller than previous.

File: src/stylesheets/components/_breadcrumb.scss [hunk 3]
```diff
@@ -7,14 +7,13 @@ $breadcrumb-context: "Breadcrumb";
...
-  #{units($breadcrumb-gap)} + #{$gap-inline-spacer}
-);
+$breadcrumb-icon-height: 2.5ex;
+$breadcrumb-icon-display-height: 2ex;
+$breadcrumb-icon-spacing: 0;
 
 $icon-breadcrumb-separator: (
-  "name": "angle-arrow-right",
-  "svg-height": 64,
+  "name": "usa-icons/navigate_next",
```
[the remark this answers is not in the document]
Comment (mejiaj) [reply src/stylesheets/components/_breadcrumb.scss:15]: image

File: src/stylesheets/elements/typography/_links.scss [hunk 4]
```diff
@@ -7,9 +7,15 @@
...
 .usa-link--external {
-  @include external-link(external-link, external-link-hover);
+  @include external-link(
+    usa-icons-bg/launch--blue-60v,
+    usa-icons-bg/launch--blue-70v
+  );
 
   &.usa-link--alt {
-    @include external-link(external-link-alt, external-link-alt-hover);
+    @include external-link(
```
[the remark this answers is not in the document]
Comment (thisisdano) [reply src/stylesheets/elements/typography/_links.scss:16]: Probably, even though I don't think one existed before...

**Comment by thisisdano** [comment]
@mejiaj Thanks for the good feedback

File: src/stylesheets/elements/typography/_links.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (thisisdano) [reply src/stylesheets/elements/typography/_links.scss:16]: Yeah, I realize I forgot to show the external link icons in the description

File: src/stylesheets/elements/form-controls/_date-picker.scss (hunk 2 above)
Comment (thisisdano) [reply src/stylesheets/elements/form-controls/_date-picker.scss:48]: I can't reproduce this one

File: src/stylesheets/components/_breadcrumb.scss (hunk 3 above)
[the remark this answers is not in the document]
Comment (thisisdano) [reply src/stylesheets/components/_breadcrumb.scss:16]: Fixed

File: src/stylesheets/elements/form-controls/_combo-box.scss (hunk 1 above)
Comment (thisisdano) [reply src/stylesheets/elements/form-controls/_combo-box.scss:45]: Fixed

File: src/stylesheets/elements/typography/_links.scss (hunk 4 above)
[the remark this answers is not in the document]
Comment (thisisdano) [reply src/stylesheets/elements/typography/_links.scss:16]: FIxed

File: src/stylesheets/elements/form-controls/_date-picker.scss (hunk 2 above)
Comment (mejiaj) [reply src/stylesheets/elements/form-controls/_date-picker.scss:48]: Seems to be a Firefox only issue. The difference is only half a pixel.

**Comment by mejiaj** [comment]
We still have older icons in:
- [Checkbox and radio](https://github.com/uswds/uswds/blob/dw-update-icons/src/stylesheets/elements/form-controls/_checkbox-and-radio.scss#L85-L97)
- [File input](https://github.com/uswds/uswds/blob/dw-update-icons/src/stylesheets/elements/form-controls/_file-input.scss#L141-L176)
- [Datepicker](https://github.com/uswds/uswds/blob/dw-update-icons/src/stylesheets/elements/form-controls/_date-picker.scss#L284-L290) 
Click on year
image


I don't see any immediate alternatives for word or excel. Maybe we can create a separate issue to find and replace?

**Review by mejiaj — APPROVED** [review]

**Pull request merged**